src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.LogicApps.tests.ps1

```diff
@@ -332,5 +332,55 @@ InModuleScope Arcus.Scripting.LogicApps {
                 Assert-MockCalled Stop-AzLogicAppRun -Scope It -Times 0
             }
         }
+        Context "Resubmitting failed Logic Apps runs" {
+             It "Resubmitting failed runs from Logic App history should succeed" {
```

Comment:
The functionality also has the ability to resubmit multiple workflows, should we test on that too?

Reply:
Unit test added